testsuite_test.go

```diff
@@ -34,8 +34,16 @@ type componentFunc struct {
 	skipRender func(prev Component) bool
 }
 
-func (c *componentFunc) Render() ComponentOrHTML        { return c.render() }
-func (c *componentFunc) SkipRender(prev Component) bool { return c.skipRender(prev) }
+func (c *componentFunc) Render() ComponentOrHTML {
+	return c.render()
+}
+
+func (c *componentFunc) SkipRender(prev Component) bool {
+	if c.skipRender != nil {
+		return c.skipRender(prev)
+	}
+	return false
+}
```

Comment:
@damz could you undo this change here and instead pass a `skipRender` function in the actual test itself?

I think having this logic be implicit can be misleading / cause us to not notice bugs. Explicitness == better in our case here.